src/transaction/client.rs

```diff
@@ -183,7 +183,9 @@ impl Client {
 
     /// Request garbage collection (GC) of the TiKV cluster.
     ///
-    /// GC deletes MVCC records whose timestamp is lower than the given `safepoint`.
+    /// GC deletes MVCC records whose timestamp is lower than the given `safepoint`. We must guarantee
```

Comment:
In fact, gc() is deprecated. The current GC mechanism used by TiDB 4.0 is described in the background section in [this RFC](https://github.com/andylokandy/rfcs/blob/gc/text/2021-04-23-builtin-gc-worker.md).

Reply:
Oh, it is.